org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/extensions/contentmodel/participants/ContentModelCompletionParticipant.java

```diff
@@ -217,4 +216,10 @@ private void fillAttributeValuesWithCMAttributeDeclarations(CMElementDeclaration
 		}
 	}
 
+	private static void updateDocumentation(CompletionItem item, String documentation, ICompletionRequest request) {
+		if (StringUtils.isEmpty(documentation)) {
+			return;
+		}
+		item.setDocumentation(request.createMarkupContent(documentation, MarkupKind.MARKDOWN));

```

Comment:
shouldn't you check markdown is supported by the client?

Reply:
Yes, it is done in createMarkupContent